README.md

```diff
@@ -105,3 +348,113 @@ logger.addHandler(handler)
 ...
 logger.removeHandler(handler)
 ```
+
+## Getting Help
+
+If you have any issue around using the library or the product, please don't hesitate to:
+
+* Use the [documentation](https://apm.docs.thundra.io/python/integration-options).
+* Open an issue in GitHub.
+* Join our python slack channel.
+
+
+## Opening Issues
+
+For any problem you encounter while using thundra, Please feel free to contact us via github issue or our python slack channel. 
+
+When opening a new issue, please provide as much information about the environment:
+* Library version, Python runtime version, dependencies, operation system with version etc.
+* Snippet of the usage.
+* A reproducible example can really help.
+
+The GitHub issues are intended for bug reports and feature requests.
+
+
+## All Environment Variables
+
+Following table shows all environment variables of [No-Code Change Tracing](#no-code-change-tracing) and their [In-Code Configuration Tracing](#in-code-configuration-tracing) equivalent.
+
+
+| Environment Variable Name                                        |  [In-Code Configuration Tracing](#in-code-configuration-tracing) Name     |
+|:-----------------------------------------------------------------|:--------------------------------------------------------------------------|
+| THUNDRA_APIKEY                                                   | thundra.apikey                                                            |
+| THUNDRA_DEBUG_ENABLE                                             | thundra.agent.debug.enable                                                |
```

Comment:
except `THUNDRA_APIKEY`, all other env vars should have must start with `THUNDRA_AGENT_...`

Reply:
done.